### DISCUSSION TITLE
Appium inspector does not split text elements inside nested navigator

### DISCUSSION THREAD

**Discussion by ivankdev** [react-navigation/react-navigation#9849] [Q&A]
React-Native app with + react-navigation-v5

After adding a screen into nested stack navigator can see picture like bellow on screenshots.

![Screenshot 2021-08-20 at 16 38 37](https://user-images.githubusercontent.com/37651838/130247042-afb2a4ad-5742-4660-9aec-7a09b4fe5f7b.png)

<img width="836" alt="Screenshot 2021-08-20 at 16 39 56" src="https://user-images.githubusercontent.com/37651838/130247254-b580f048-17c4-444e-8bd1-ed816d9ab779.png">

Appium cannot split text elements inside view even with accessible={false}.

If I keep this screen out of nested stack on the level of root one - it works. Should I set some configs on navigation side or appium side?

**Comment by vkukade-altir** [comment]
facing same issue.

**Comment by fawaz-ahmed** [comment]
This is probably happening due to pager view which top tabs is using. Similar issue reported there https://github.com/callstack/react-native-pager-view/issues/1033

Can we just choose to have a react native default View inside the tab ?
